pkg/status/templates/header.tmpl

```diff
@@ -86,6 +86,15 @@ NOTE: Changes made to this template should be reflected on the following templat
     {{ $key }}: {{ $value }}
   {{- end }}
 
+{{- if eq .config.fips_enabled true }}
+
+  FIPS proxy
+  ==========
+    FIPS proxy is enabled. All communication will be routed to a local FIPS proxy:
```

Comment:
(also, it'd be ideal if we could explain in a very short sentence that there's no guarantee that the proxy is actually FIPS compliant unless the user has set up the FIPS proxy according to our documentation, since in theory users could set up a non-FIPS proxy as well... But we can figure this out later unless you have a good idea of how to communicate this)

Reply:
This is planned in a second PR where we add the documentation to the config_template.